src/tests/posix/sys/timerfd/timerfd_tests.c

```diff
@@ -79,7 +82,7 @@ TEST_CASE("timerfd set and get (one-time timer)") {
 	test_assert_true(timespec_is_zero(&curr_value.it_interval));
 
 	// second gettime: timer expired
-	usleep(USEC_PER_SEC / 4);
+	usleep(USEC_PER_SEC * 2);
```

Comment:
It seems its really so bad with our timers because the timer was set up at 1 second, and now you check whether it expired only after 2 seconds. Why? Because the timer didn't expire after 1 sec like it was assumed?

Reply:
Ok, I created trello card.